src/shared/components/Input/component.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {
+  InputRootContainer,
+  InputContainer,
+  InputWrapper,
+  InputValidationMessageWrapper,
+  InputGroupButton,
+} from './styles';
+import { SendIcon } from '../../icons';
+
+export type InputProps = React.HTMLAttributes<HTMLInputElement> & {
+  inputType: string;
+  validationState: boolean;
+  validationMessage: string;
+  withSend?: boolean;
```

Comment:
I am not sure about `withSend` naming, it looks a bit confusing for me. 

Let's call it like `includeSendButton` to be more clear.

Reply:
done